lib/node_modules/@stdlib/stats/base/cuminabs/test/test.cuminabs.js

```diff
@@ -123,6 +191,20 @@ tape( 'the function returns a reference to the output array', function test( t )
 	t.end();
 });
 
+tape( 'the function returns a reference to the output array (accessor)', function test( t ) {
+	var out;
+	var x;
+	var y;
+
+	x = toAccessorArray([ 1.0, 2.0, 3.0, 4.0, 5.0 ]);
+	y = toAccessorArray([ 0.0, 0.0, 0.0, 0.0, 0.0 ]);
```

Comment:
```suggestion
	x = toAccessorArray( [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
	y = toAccessorArray( [ 0.0, 0.0, 0.0, 0.0, 0.0 ] );
```

Use consistent spacing.